posts/hybrid-reference-profiles/index.qmd

```diff
@@ -0,0 +1,70 @@
+---
+title: "Creating hybrid reference profiles for InSituType"
+author:
+  - name: Patrick Danaher
+    orcid: 0000-0002-2844-5883
+    affiliations:
+      - ref: nstg
+      - ref: patrickjdanaher
+date: "2024-08-21"
+date-modified: "2024-08-21"
+categories: [cell typing]
+draft: false
+image: figures/heatmap_tile.png
+---
+
+
+## Background
+
+The InSituType cell typing algorithm relies on a "reference matrix" to perform supervised or semi-supervised cell typing.
+A reference matrix gives the expected gene expression profile of each cell type in a tissue;
+these are usually derived from previous scRNA-seq or spatial transcriptomics experiments. 
```

Comment:
A common question that we get is 'how do I make a reference profile from a scRNA-seq dataset?'. This would be a great place to address that question - this could be with basic code to derive a profile, it could use the new `getRNAprofiles` function in InSituType, or it could be a link to InSituType documentation, if you have further descriptions there. When putting together this recommendation, two key questions to address are (1) should the input RNA data be normalized? If yes, how? and (2) if you recommend using the InSituType function, what should be used for the background term for scRNA-seq data?